### PR TITLE
Cu 864dzdwf0 projects page iguoliveira

### PR DESCRIPTION
**Pull request by iguoliveira**
##### Description of changes

- [x] Why (Why did you make this pull request): Created the entire project page with logic + kanban initial logic. Hurry to approve this for the next week meeting to show the results

- [x] Brief description of the changes: Project page creation and kanban page. All logic
- [x] Possible breakouts (Possible places where things could break): Project page and kanban. All w/o try/catch kekw

### CODE REVIEW

File: apps/web-planner/src/pages/project/[id]/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                        })}
+                        <NewColumn sendData={handleSubmit} />
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    )
+}
+
```
Comment (vinicius-prates) [inline apps/web-planner/src/pages/project/[id]/index.tsx:64]: Looking good, clean code

**Review by vitormiura — APPROVED** [review]
resolved conflicts rn, looks good to me :+1:

**Comment by iguoliveira** [comment]
I'll need to fix somethings
Like a bug in project creation, whose use the same kanban to all projects

Later, it'll be ready to review!
😄

**Review by viktormarinho — APPROVED** [review]

**Comment by viktormarinho** [comment]
Anyone that needs help regarding something i did in this PR, just call me

**Pull request merged**